aspnetcore/security/authentication/social/social-without-identity.md

```diff
@@ -53,6 +53,18 @@ Test the app's authentication configuration by applying the [[Authorize]](xref:M
 
 :::code language="csharp" source="social-without-identity/samples/6.x/SocialWithoutIdentitySample/Pages/Privacy.cshtml.cs" id="snippet_Class" highlight="1":::
 
+## Save the access token
+
+<xref:Microsoft.AspNetCore.Authentication.RemoteAuthenticationOptions.SaveTokens%2A> defines whether access and refresh tokens should be stored in the <xref:Microsoft.AspNetCore.Http.Authentication.AuthenticationProperties> after a successful authorization. `SaveTokens` is set to `false` by default to reduce the size of the final authentication cookie.
```

Comment:
This heading and introductory paragraph are taken straight from [Persist additional claims and tokens from external providers in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/additional-claims?view=aspnetcore-6.0#save-the-access-token), which already covers `SaveTokens` when used with Identity.